manifests/m/Microsoft/AzureInformationProtection/2.18.26.0/Microsoft.AzureInformationProtection.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.AzureInformationProtection

```

```suggestion
PackageIdentifier: Microsoft.Azure.InformationProtection
```
This should fall under the Azure related packages